packages/shared/src/lib/featureManagement.ts

```diff
@@ -41,8 +42,9 @@ const feature = {
     desktop: 25,
   }),
   pricingIds: new Feature('pricing_ids', {
-    pri_01jbsccbdbcwyhdy8hy3c2etyn: 'monthly',
-    pri_01jbscda57910yvwjtyapnrrzc: 'yearly',
+    pri_O1jdkx6sakk5kb6p586nejnqam: PlusPriceType.Monthly, // early adopter
+    pri_01jbsccbdbcwyhdy8hy3c2etyn: PlusPriceType.Monthly, // plus monthly
+    pri_01jbscda57910yvwjtyapnrrzc: PlusPriceType.Yearly,
```

Comment:
Please revert this before merging

Reply:
Aren't those the prod ones?

Reply:
The ones you're adding are dev ones?
Basically this shouldn't need any changes for the default that's now live.

Reply:
Montly/Yearly are the same as before, `early adopter` is the one used in prod ✔️

Reply:
Yes but it's an experiment you are now adding it as a default

Reply:
Yes mb, removed! ✔️